src/index.html

```diff
@@ -2,16 +2,153 @@
+              autocomplete="off"
+            >
+          </div>
+          <div class="form-field">
+            How old are You?
+            <input
+              type="number"
+              min="1"
+              max="100"
+              value="12"
+              name="age"
+            >
+          </div>
+          <div class="form-field">
+            Full date of birth:
+            <input type="date">
+          </div>
+          <label class="form-field">
+            I accept the term of the agreement
+            <input type="checkbox"name="term">
+          </label>
+        </fieldset>
+        <fieldset>
+          <legend>Registration</legend>
+          <div class="form-field">
+            E-mail:
+            <input
+              type="email"
+              name="mail"
+              placeholder="email@example.com"
+              required
+            >
+          </div>
+          <div class="form-field">
+            Password:
+            <input
+              type="password"
+              name="password"
+              required
+            >
+          </div>
+        </fieldset>
+        <fieldset>
+          <legend>An interesting fact about you!</legend>
+          <div class="form-field">
+            <label>
+              Do you love cats?
+              <input
+                type="radio"
+                name="answer"
+                value="Yes"
+              >
+              Yes
+            </label>
+            <label>
+              <input
+                type="radio"
+                name="answer"
+                value="No"
+              >
+                No
+            </label>
+          </div>
+          <div class="form-field">
+            What is your favorite color?
+            <input
+              type="color"
+              name="color"
+              value="#000"
+            >
+          </div>
+          <div class="form-field">
+            What time do you go to bed?
+            <input
+              type="time"
+              step="1"
+              name="appt"
+            >
+          </div>
+          <div class="form-field">
+            <label for="cars">
+              What are your favorite brand of cars?
+            </label>
+            <select name="cars" multiple>
+              <option value="bmw">BMW</option>
+              <option value="audi">Audi</option>
+              <option value="lada">Lada</option>
+            </select>
+          </div>
+          <div class="form-field">
+            How do you rate our work?
+            <input
+              type="range"
+              id="range"
+              name="range"
+            >
+          </div>
+        </fieldset>
+        <fieldset>
+          <legend>Additional info:</legend>
+          <div class="form-field">
+            Comments:
+            <textarea
+              name="feedback"
+              rows="2"
+              autocomplete="off"
+              >
+            </textarea>
+          </div>
```

Comment:
![image](https://user-images.githubusercontent.com/99331793/187884480-0c497091-88c7-4ca2-99a1-cad14059938d.png)

spaces inside of text area
```suggestion
          <div class="form-field">
            Comments:
            <textarea
              name="feedback"
              rows="2"
              autocomplete="off"
              ></textarea>
          </div>
```